---
title: Quantative Polymerase Chain Reaction (qPCR)
subtitle: Lab Report - Session 6
code-fold: true
---

# Objective

The objective of the experiment is to determine the concentration of an unknown experimental DNA sample using quantitative PCR (qPCR) by calculating the concentration of DNA by referencing known control sample.

# Background

## Quantitative PCR (qPCR) {#sec-qpcr}
qPCR quantifies the amplification of DNA in real-time by measuring the fluorescence of primers successfully bound, and released, from their complementary DNA strands. qPCR is distinct from PCR in that it allows for the measurement of DNA concentrations as opposed to qualitative analysis of the gel sample against a base pair ladder.[^edvotek]

qPCR contains a series of variables which provide useful metrics when analyzing the amplification of DNA. A key variable is the *quantification cycle (Cq)*, or the cycle number at which a sample's fluorescence reaches a preset threshold.

Lower Cq values indicate a higher initial concentration of target DNA. This gives an important measure as to the reliability of the PCR amplification process to produce the Gene-of-Interest.

## Applications of qPCR {#sec-applied-qpcr}

qPCR has several applications which relies upon its ability to provide quantified analyses on a targeted DNA sequence within the sample, and qPCR's melting-curve program for verifying its accuracy.

**Gene Expression Analysis**: qPCR is widely used to study changes in gene expression levels by quantifying specific mRNA transcripts.

**Disease Diagnosis and Management**: qPCR is valuable for diagnosing and monitoring diseases, particularly viral infections by quantifying viral DNA or RNA to assess the viral load in patients.

**Food Testing**: qPCR can be applied in food safety and quality control to detect and quantify genetically modified organisms (GMOs) in food products.

# Results

## Scatter plot of Cycle Threshold by Log DNA Concentration 

![Excel plot of the qPCR results data with the x/y-axis being the Ct value, Log DNA Concentration, and linear regression equation with R2 value.](images/exl_results.png){#fig-scatter-results}

## Melting curve of qPCR Samples

![Edvotek Thermocycler Melt Curve Plot of qPCR sample results measuring the derivative of fluorescence levels across temperature.](images/results_curve.png){#fig-mcurve-results}

## Calculated concentration of DNA from Unknown Samples 

![Excel table of calculated concentration of DNA from experimental samples provided in the lab. Where Wells A2:D2 are known concentrations, while E2:H2 are unknown concentrations.](images/calc_qpcr.png){#fig-calc-qpcr}

- Data for Well Positions E2:H2 was calculated using the m and b values from @fig-scatter-results where m = -0.1753 and b = 3.1369.

- The equation for calculating nanograms of DNA per micro-liter within the unknown samples was $R_o=10^{(C_q-b)/m}$. 

- Wells A2:D2 were known concentrations of nanograms DNA per micro-liter in consecutively increasing ratios from 1:1 to 1:1000.

# Interpretation & Discussion

## $R^2=0.085$ value from @fig-scatter-results suggests an inefficient qPCR amplification compared to the $R^2=0.98$ standard

Given the lab manuals statement that an "*Efficient qPCR reactions will have an $R^2$ value of 0.98*"[^edvotek], analysis of the linear regression provided by the above calculations suggests and inefficient amplification result from the qPCR reaction.

This implies the starting concentration of DNA for primer binding was considerably lower, and the Cq values listed in @fig-calc-qpcr are higher than the efficiency standard.

## Analysis of Edvotek ThermoCycler Melt Curve results plotted in @fig-mcurve-results {#sec-mc-curve}

### Experimental Samples: {#sec-exp-samples}

The experimental samples contained staggered melting temperatures, and/or dual peaks. The first peaks within the experimental samples appeared pre-65 degrees celsius, suggesting a considerable quantity of amplicons across the experimental samples.

Potential causes for this phenomena imply issues with the ligation process binding the primers to the DNA samples. However, the elevated base-level of fluorescence across all samples also points towards premature activation of the fluorescent tags as further elucidated in @sec-kn-fl.

#### Targets 5, 7 {#sec-targets-5-7}
Targets 5, 7 had peaks pre-80 degrees celsius suggesting they contained a large quantity of amplicons when compared again the study question references provided in @fig-curve-2. These curves were also notable because they were almost twice as fluorescent as the maximum level achieved by the samples with known concentrations.

#### Targets 6, 8 {#sec-targets-6-8}

Targets 6, 8 had one peak post-85 degrees. These melting points suggest the sample did *not* contain amplicons when compared against the reference images provided within @fig-curve-2. While the elevated base-level of fluorescence suggests free-floating fluorescent probes within the sample, the curves post-85 degrees does suggest validity within the accuracy of the sample.

### Known Concentration Samples {#sec-known-samples}

The samples within Targets 1 through 4 were the samples with a known concentration of DNA. These targets were used to calculate the concentration of the unknown samples in @sec-calc-unknown. 

#### Fluorescence levels of the known samples {#sec-kn-fl}

Of note, none of the samples with known concentrations were able to reach a fluorescence level above the experimental samples. Potential cause for this phenomena would be activation of the fluorescent dye within the sample by the lab's ambient light interacting with the samples before reaching its melting point prematurely within the thermocycler.

#### All Known Sample exhibited a dip in fluorescence before their peaks post-85 degrees celsius {#sec-known-samples-dip}

An unexplained artifact within @fig-mcurve-results is the dip in fluorescence around the 80 degrees celsius mark. Potential explanations for the dip would be similar to those outline in @sec-kn-fl.

## Calculated concentrations of unknown samples yielded infinitesimally tiny values {#sec-calc-unknown}

Given the variability and low fluorescence of the known samples as described in @sec-known-samples, the accuracy of these calculations is considerable despite the validity of the calculations involved. 

A secondary experiment would be required to gain greater signal from the known control samples to ensure a more accurate reading of the DNA concentration of the unknown samples.

# Study Questions 
## What are the three steps in a polymerase chain reaction and what does each accomplish?

PCR is a three step process composed of:

1. **Denaturation** occurs at **94°C**. During denaturation, the double-stranded DNA molecule is decomposed into two single strands.[^lehninger]

2. **Annealing** occurs between **40-65°C**. During annealing, Taq polymerase primers bind to complementary regions on the single-stranded DNA templates. Primers flank the target DNA sequence to be amplified, acting as the starting point for polymerase to synthesize new DNA.

3. **Extension** occurs at **72°C**. During extension, a heat-stable DNA polymerase called Taq polymerase adds nucleotides to the 3' end of the primers, synthesizing a new strand of complementary DNA.

These three steps are repeated cyclically, exponentially increasing the amount of the target DNA sequence in the sample. 

## How much does target DNA increase during each PCR cycle under ideal conditions? What are some factors that can dampen this rate of increase?

Under ideal conditions, the target DNA in a polymerase chain reaction (PCR) creates an exponential increase in the amount of the specific DNA sequence being targeted. 

## How is amplified DNA visualized in regular PCR? How is it visualized in qPCR? What information can be gained from each approach?

### Regular PCR
In regular PCR, amplified DNA is visualized after the PCR process is complete. The PCR products are separated and visualized using gel electrophoresis, stained, and compared alongside a DNA ladder to compare the speed of migration.

- **Presence or Absence of the Target Sequence**: If a band of the expected size is observed on the gel, it indicates that the target DNA sequence was present in the original sample and was successfully amplified.

- **Relative Amount of Amplified DNA**: The intensity of the band on the gel can give a rough indication of the amount of DNA amplified when compared against the Base Pair Ladder.[^edvotek]

### Quantitative PCR (qPCR)
In qPCR, the amplification of DNA is monitored *in real time*, during the PCR process. This is achieved by using fluorescent dyes or probes that bind to the DNA and emit fluorescence as the DNA is amplified.

- **Measuring Target DNA**: qPCR allows for the precise measurement of the initial amount of target DNA in the sample. This is possible because the fluorescence signal is directly proportional to the amount of DNA present.

- **Melting Curve Analysis**: The melting curve analysis, which measures the temperature at which the double-stranded DNA product melts into single strands, can provide information about the specificity of the PCR reaction and help identify the presence of non-specific products or primer dimers. 

## Compare and contrast relative quantification and absolute quantification in the context of qPCR.

### Relative Quantification
Relative Quantification aims to determine the fold change in target gene expression between different samples. 

**Method:** Relative Quantification involves comparing the expression of the target gene to housekeeping gene by measuring the difference between the *threshold cycle (Ct)* values between the two. The Ct value represents the PCR cycle where the fluorescence signal crosses a defined threshold measuring the significant increase in amplified DNA.[^edvotek]

### Absolute Quantification
Absolute Quantification determines the number of copies of the target DNA sequence in a sample. 

**Method:** Absolute Quantification requires the use of a standard curve. This curve is generated using a series of known concentrations of a DNA standard that corresponds to the target sequence. 

By plotting the Ct values of the standard samples against their known concentrations on a logarithmic scale to establish a linear relationship. This standard curve allows for the conversion between the Ct value of an unknown sample into its corresponding DNA concentration.

## What occurs to DNA during a melting curve program? And what do the results say about the PCR reaction? {#sec-q-melting-curve}

A melting curve program assess the quantity of *amplicons* which occured during the qPCR process. Amplicons occur when primers bind to each other, as opposed to the Gene-of-Interest. 

- **Single Peak:** A single, well-defined peak in the melting curve suggests that the PCR reaction amplified a single, specific product. This indicates that the primers used in the reaction were highly specific for the target sequence.

- **Multiple Peaks:** Multiple peaks in the melting curve suggest the presence of non-specific products, such as primer dimers (primers annealing to each other) or amplification of unintended sequences.

## Why might a researcher decide to use qPCR over traditional PCR? {#sec-on-pcr}

qPCR's most significant advantage is its ability to *quantify* the amount of DNA present in a sample, while traditional PCR only provides a *qualitative* assessment.

qPCR monitors DNA amplification in real-time using fluorescent dyes or probes. As the amount of DNA increases during each PCR cycle, the fluorescence signal also increases, allowing researchers to directly measure the amount of DNA being amplified.

## The alternative melting curve (Figure 2) looks different from the melting curve (Figure 1). What would cause the melting curve found in Figure 2 and why?

![Melting curve without amplicons as evinced by the concentration of the samples around a singular peak in right-hand curve.](images/fig1_melting_curve.png){#fig-curve-1}

![Melting curve with primer dimer amplicons evinced by the staggered, low melting temperatures and the blue sample's dual peaks.](images/fig2_melting_curve.png){#fig-curve-2}

As mentioned in @sec-q-melting-curve, the two peaks in @fig-curve-2 evince the presence of amplicons within the sample. Amplicons occur when the primers bind to other free-floating primers as opposed to the Gene-of-Interest, this creates the two melting curves seen above.

# References
[^edvotek]: Edvotek. (2008-2024). Biotechnolgoy Cirriculum Modules. Section 6 - "Quantative PCR (qPCR)"
[^lehninger]: David L. Nelson; Michael M. Cox. (2021). Lehninger Principles of Biochemistry. "*Double-Helical DNA and RNA Can Be Denatured*" pg 297.
[^adesh]: Suchit Adesh. (2024). Conversations on the Nature of qPCR
